packages/driver/patches/jquery+3.4.1.dev.patch

```diff
@@ -72,10 +72,32 @@ index 411a859..0000000
 -});
 -```
 diff --git a/node_modules/jquery/dist/jquery.js b/node_modules/jquery/dist/jquery.js
-index 773ad95..9e6076f 100644
+index 773ad95..843bf56 100644
 --- a/node_modules/jquery/dist/jquery.js
 +++ b/node_modules/jquery/dist/jquery.js
-@@ -6534,69 +6534,100 @@ function boxModelAdjustment( elem, dimension, box, isBorderBox, styles, computed
+@@ -1100,21 +1100,6 @@ setDocument = Sizzle.setDocument = function( node ) {
+ 	docElem = document.documentElement;
+ 	documentIsHTML = !isXML( document );
+ 
+-	// Support: IE 9-11, Edge
+-	// Accessing iframe documents after unload throws "permission denied" errors (jQuery #13936)
+-	if ( preferredDoc !== document &&
```

Comment:
to avoid potentially breaking Edge, should the patch be closer to what jQuery did as of 3.7.1? 

```
if ( documentElement.msMatchesSelector &&

		// Support: IE 11+, Edge 17 - 18+
		// IE/Edge sometimes throw a "Permission denied" error when strict-comparing
		// two documents; shallow comparisons work.
		// eslint-disable-next-line eqeqeq
		preferredDoc != document &&
		( subWindow = document.defaultView ) && subWindow.top !== subWindow ) {

		// Support: IE 9 - 11+, Edge 12 - 18+
		subWindow.addEventListener( "unload", unloadHandler );
	}
```

Reply:
Ah, I read this initial comment at Edge 12-18 which was so low, but I see now this is Edge 18+. This is what I was initially doing, but removing it seemed easier, so let me revisit this.

Reply:
The more I think about this, why doesn't it just say Edge 12+???